Revise Installing Cygwin on Windows

ari-gold opened the pull request:
* Expand use cases to include running openssl and installing and using Terminus
* Remove excess images
* Add keywords to front matter
* Incorporate edits from @rachelwhitton and @nataliejeremy

Closes #807

rachelwhitton commented:
@ari-gold great update! Thanks for the issue and PR! I added a screenshot because I couldn't find one on this branch or in master for cygwin, can you +1 my revision or provide an alternate screenshot if I missed the mark? 

Thanks again!


+1

ari-gold commented:
Thanks @rachelwhitton! Whoops I forgot to include the image. Added it and incorporated your edits. Do you think it needs annotation/red rectangle?

@nataliejeremy Can you review as well?

rachelwhitton commented:
nice @ari-gold! I think it's better without annotations. The description you provided has enough details 

+1
